assets/js/components/ErrorText.js

```diff
@@ -32,7 +32,7 @@ import { sprintf, __ } from '@wordpress/i18n';
  */
 import { sanitizeHTML } from '../util';
 
-function ErrorText( { message, reconnectURL } ) {
+function ErrorText( { message, reconnectURL, children } ) {
```

Comment:
@sashadoes Since `defaultProps` will be deprecated, let's set the default values for the props here itself and remove the `defaultProps` property.
```suggestion
function ErrorText( { message, reconnectURL = '', children = null } ) {
```